source/reference/minio-server/minio-server.rst

```diff
@@ -2216,9 +2216,20 @@ For example, the following command set two distinct Object Lambda webhook endpoi
    :class: copyable
 
    export MINIO_LAMBDA_WEBHOOK_ENABLE_myfunction="on"
-   export MINIO_LAMBDA_WEBHOOK_ENDPOINT_myfunction="http://webhook-1.example.net
+   export MINIO_LAMBDA_WEBHOOK_ENDPOINT_myfunction="http://webhook-1.example.net"
    export MINIO_LAMBDA_WEBHOOK_ENABLE_yourfunction="on"
-   export MINIO_LAMBDA_WEBHOOK_ENDPOINT_yourfunction="http://webhook-2.example.net
+   export MINIO_LAMBDA_WEBHOOK_ENDPOINT_yourfunction="http://webhook-2.example.net"
+
+.. envvar:: MINIO_LAMBDA_WEBHOOK_ENABLE
+
+   Specify ``"on"`` to enable publishing :mc:`minio server` logs to the Object Lambda
+   webhook endpoint for the specified function.
```

Comment:
Is this just server logs that get output?

Reply:
Dropped the logs bit, as there don't appear to be any involved. Just the request/response for the handler.